models/response.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+// Model
+const responseSchema = mongoose.Schema({
```

Comment:
Why did you change this to a const?

Reply:
Sorry, should have called this out explicitly. Schemas and models never change after being defined, so it made more sense to define as a `const` to ensure that they don't get changed accidentally.